Automattic/jetpack
# Add tooltip for installed with no videos

## Proposed changes:

Add tooltip for the following states of the videopress card
* Inactive with videos
* Active with no videos
* Active with a plan
* Active without a plan

These tooltips will have to be adjusted when we add the comparison data and when we add context switching between "last 30 days" and "all-time" stats. That will be handled in those individual PRs

### Other information:


## Jetpack product discussion

P2: p1HpG7-rb7-p2

## Does this pull request change what data or activity we track or use?

Yes, there are tracks for all the tooltips onClick, as well as a track when the user clicks a link within one of the tooltips

## Testing instructions:

To see the tooltip copy you can look here: tCejO24bAGT2ME7iYEfmuG-fi-6392_3262

1. Create a Jurassic ninja site with sandbox access and checkout this branch on Jetpack (you may want to test this on your local environment, the stats are cached in a transient and it could be useful to comment that out when testing)
2. Go to My Jetpack. There should be no tooltips for VideoPress when the product is unowned and no videos are uploaded
![image](https://github.com/user-attachments/assets/591cad29-7051-44af-ac64-808a218b48d5)
3. Now add 1 or more videos to your site and go back to My Jetpack. You should now see a tooltip icon and when you open it, you should see the correct amount of videos in the content, as well as a link to some documentation
![image](https://github.com/user-attachments/assets/40e48d94-6795-4748-bc9b-4eea4c71e0f5)
4. Make sure the `hosting` link goes to the right place (https://jetpack.com/support/jetpack-videopress/the-jetpack-videopress-dashboard/#manage-local-videos)
5. Now activate the product and connect your user account
6. To mock some randomized data, go to uggcf%3N%2S%2Suers.yv%2S%3Suggcf%3N%2S%2Sbcratebx.n8p.pbz%2Sfbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qwfba%2Qncv%2Qfgngf%2Qivqrb%2Qcynlf%2Qi1%2Q1%2Qraqcbvag.cuc%3Se%3Q210o65nn%26zb%3Q1199%26sv%3Q37%2350-og and replace all those zeros with something like rand(0, 100). You can pick whatever numbers you'd like if you wanted to test out smaller or bigger numbers.
7. Make sure you only have 1 video and look at the tooltips. They should both be in the context of a site without a plan and only 1 video
![image](https://github.com/user-attachments/assets/9d82cd8c-3196-4bdd-8ca0-2c65016df6a9)
![image](https://github.com/user-attachments/assets/3bdcf546-672e-4840-954d-392c639b821d)
8. Now purchase a VideoPress plan and go back to My Jetpack. The tooltip related to views should now be updated.
![image](https://github.com/user-attachments/assets/cdec198a-4673-4310-83d5-aa832bb43baa)

Bonus: Test with an active plugin with 0 stats and 1 video. Ensure that we do not imply they are currently gathering views on their video
![image](https://github.com/user-attachments/assets/f7b93d88-9e78-4dd4-a752-3912ea3be13d)
